lib/src/chip.rs

```diff
@@ -373,6 +375,7 @@ impl AsRawFd for Chip {
 
 /// The publicly available information for a GPIO chip.
 #[derive(Clone, Debug, Default, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize), serde(rename_all = "camelCase"))]
```

Comment:
Using "camelCase" is helpful for interacting with JavaScript/TypeScript programs that assume camel case key names for objects.

Reply:
Given you are generating JSON, camelCase is the way to go.